crates/core/src/rpc/surfnet_cheatcodes.rs

```diff
@@ -746,6 +749,66 @@ pub trait SurfnetCheatcodes {
         config: Option<ResetAccountConfig>,
     ) -> Result<RpcResponse<()>>;
 
+    /// A cheat code to export a snapshot of all accounts in the Surfnet SVM.
+    ///
+    /// This method retrieves the current state of all accounts stored in the Surfnet Virtual Machine (SVM)
+    /// and returns them as a mapping of account public keys to their respective account snapshots.
+    ///
+    /// ## Parameters
+    /// - `config`: An optional `ExportSnapshotConfig` to customize the export behavior. The config fields are:
+    ///     - `includeParsedAccounts`: If true, includes parsed account data in the snapshot.
+    ///     - `filter`: An optional filter config to limit which accounts are included in the snapshot. Fields include:
+    ///         - `includeProgramAccounts`: A list of program IDs to include accounts for.
```

Comment:
Copy pasta, this is a `bool`?
We know that at some point we'll differentiate local vs fetched, should we already introduce this distinction?  It could be an `enum` instead - `all` (supported with this version), (`remoteOnly` to be supported)